Modules/xASL_module_Population.m

```diff
@@ -201,75 +201,84 @@
 %% 7    ROI statistics
 if ~x.mutex.HasState(StateName{8})
     
-    % Default & Fallback for Atlases
+    % Default atlases/ROIs
     if ~isfield(x.S,'Atlases')
         x.S.Atlases = {'TotalGM','DeepWM'}; % Default
     end
-
+    
+    % Default datatypes
+    if ~isfield(x.S,'DataTypes')

```

Comment:
Probably add
|| isempty(x.S.DataTypes)
just to be sure.

Reply:
Done